central/processlisteningonport/datastore/datastore_impl_test.go

```diff
@@ -2455,19 +2648,19 @@ func (suite *PLOPDataStoreTestSuite) TestRemoveOrphanedPLOPsByProcesses() {
 			}
 
 			suite.NoError(suite.indicatorDataStore.AddProcessIndicators(suite.hasWriteCtx, c.initialProcesses...))
-			countFromDB, err := suite.indicatorDataStore.Count(suite.hasReadCtx, nil)
+			countFromDB, err := suite.indicatorDataStore.Count(suite.hasAllCtx, nil)
 			suite.NoError(err)
 			suite.Equal(len(c.initialProcesses), countFromDB)
 
 			err = suite.store.UpsertMany(suite.hasWriteCtx, c.initialPlops)
 			suite.NoError(err)
-			plopCount, err := suite.store.Count(suite.hasReadCtx, search.EmptyQuery())
+			plopCount, err := suite.store.Count(suite.hasAllCtx, search.EmptyQuery())
 			suite.NoError(err)
 			suite.Equal(len(c.initialPlops), plopCount)
 
 			suite.datastore.PruneOrphanedPLOPsByProcessIndicators(suite.hasAllCtx, orphanWindow)
 
-			plopCount, err = suite.store.Count(suite.hasReadCtx, search.EmptyQuery())
+			plopCount, err = suite.store.Count(suite.hasAllCtx, search.EmptyQuery())
```

Comment:
Would `suite.hasReadCtx` be enough to count the items in the PLOP store ?

Reply:
Yes. Reverted.